scripts/bootstrap-aio.yml

```diff
@@ -33,6 +33,20 @@
     rpco_deploy_ceph: "{{ lookup('env', 'DEPLOY_CEPH') }}"
     rpco_deploy_hardening: "{{ lookup('env', 'DEPLOY_HARDENING') }}"
 
+- name: Enable trusty-backports
+  hosts: all
+  tasks:
+    - name: Modify sources.list
+      lineinfile:
```

Comment:
I expect that using the copy module with the content argument will be more efficient than running a task using with_items. Each item results in a script copied to the host to be executed, whereas the copy module with content argument will only result in one action against the target host.